scripts/machine-launch-script.sh

```diff
@@ -166,13 +166,15 @@ set -o pipefail
 
         # see https://conda-forge.org/docs/user/tipsandtricks.html#multiple-channels
         # for more information on strict channel_priority
-        "${DRY_RUN_ECHO[@]}" $SUDO "$CONDA_EXE" config --system --set channel_priority strict
+        "${DRY_RUN_ECHO[@]}" $SUDO "$CONDA_EXE" config --system --set channel_priority flexible
```

Comment:
@timsnyder-siv Do you see any issues with making this a bit more flexible?

Reply:
Since we're installing 'miniforge3' and it doesn't have the 'Defaults' channel that has known conflicts with conda-forge, this should be okay.  It does make it much more complicated to determine which channel any given package is going to come from but I see why you need to have this with the libdwarf-dev

Reply:
@abejgonzalez see also my comment on the chipyard conda PR at https://github.com/ucb-bar/chipyard/pull/1163/files#r953157747. Since you guys are just saying 'install conda' and point to the Anaconda Inc instructions, this loosening might be problematic for anyone who also has 'defaults' in their conda configuration

Reply:
I've addressed this in the CY PR you mentioned. In any case, #1180 will double-check this by adding the `nodefaults` channel to the `conda-requirements.yaml` file.